training_maquetado/index.html

```diff
@@ -11,6 +11,11 @@
 </head>
 
 <body>
+  <div class="navbar-container"> 
+    <img class="logo-color" src="assets/wolox_logo.svg">
+    <h1 class="navbar-title">Developer Tools</h1>
```

Comment:
Use h1 just for important or destacable content, for SEO reasons try to keep as low and descriptive as possible your h1 tags, as you already has used h1 for the titles use another one for this.